gen/voms

```diff
@@ -48,27 +49,47 @@ foreach my $resourceData ($data->getChildElements) {
 		#group members one by one
 		foreach my $memberData (($groupData->getChildElements)[1]->getChildElements) {
 			my %memberAttributes = attributesToHash $memberData->getAttributes;
+			my $memberUniqueIdentifier;
 			#skip member if his status is not valid
 			next unless $memberAttributes{$A_USER_STATUS} eq $STATUS_VALID;
 			#get mail
 			my $email = $memberAttributes{$A_USER_MAIL};
 
 			#each DN of user is separate instance of user in voms
 			#skip users with no certificates
-			foreach my $subjectDN (keys %{$memberAttributes{$A_USER_CERT_DNS}}) {
+			foreach my $subjectDN (sort keys %{$memberAttributes{$A_USER_CERT_DNS}}) {
+				#set uniqueIdentifier for member (his first certificate DN+CA)
+				unless($memberUniqueIdentifier) { $memberUniqueIdentifier = $subjectDN . "---------------" .  $memberAttributes{$A_USER_CERT_DNS}{$subjectDN}; }
```

Comment:
OK. it's just cosmetics. Since it's only an ephemeral structure it's of little concern.